tests/circuit_drawer/test_grid.py

```diff
@@ -244,5 +244,7 @@ def test_replace_error_message(self):
         an uninitialized Grid is attempted."""
         grid = Grid()
 
-        with pytest.raises(AttributeError, match="Can't replace layer. The Grid has not yet been initialized."):
+        with pytest.raises(
+            AttributeError, match="Can't replace layer. The Grid has not yet been initialized."
+        ):
```

Comment:
Is this correctly blacked with `black -l 100`? I'm surprised the formatting changed, as I can't spot any other differences.

Reply:
Yes, I was also confused